.coveragerc

```diff
@@ -12,6 +10,13 @@ omit =
     conpot/core/loggers/stix_transform.py
     conpot/core/loggers/taxii_log.py
     conpot/utils/mac_addr.py
+    # New features.. Just to avoid coverage drop - before we actually write some tests
```

Comment:
you're starting to take drops in coverage personally, do you? ;)

Reply:
Hehe. No. Not really 😉